doc/nrf/doc_styleguide.rst

```diff
@@ -120,6 +202,43 @@ To link directly to a doxygen reference from RST, use the following Breathe doma
 
       :cpp:enumerator:`ENUM_VALUE <DOXYGEN_GROUP::ENUM_VALUE>`
 
+PCB names
+=========
+
+When referring to specific Printed Circuit Board elements, use capitalization.
+For example, *Button 1*, *Switch 3*, *LED 1*.
+
+If you provide the code of a specific PCB element, write it in bold: **SW3**.
+
+If you are referring to a generic PCB element, do *not* use capitalization: *button*, *switch*, *LEDs*.
+
+
+Tables
+======
+
+Follow Zephyr's :ref:`zephyr:doc_guidelines` for tables.
+However, do not add table titles to describe the table.
```

Comment:
```suggestion
Do not add table titles to describe the table.
```
With the "however" it sounds like we're diverting from Zephyr's guidelines - but we aren't. Zephyr doesn't say anything about table captions, their style is just as inconsistent as ours. ;)